project_status/tests/test_status.py

```
@@ -0,0 +1,30 @@
+from odoo.tests.common import SavepointCase
+
+
+class TestProjectStatus(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
```

Ruff (0.11.9)

Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)